src/test/java/fr/pilato/elasticsearch/crawler/fs/test/integration/AbstractITCase.java

```diff
@@ -244,6 +244,8 @@ public static SearchResponse countTestHelper(final String indexName, String quer
                 sr.setFields(fields);
             }
 
+            sr.setSize(0);
```

Comment:
I'd not set that. I think it can have bad impact on some tests which read the response sent by elasticsearch later on.